lib/identity/stateTransitions/AbstractIdentityStateTransition.js

```diff
@@ -4,7 +4,40 @@ const AbstractStateTransition = require('../../stateTransition/AbstractStateTran
  * @abstract
  */
 class AbstractIdentityStateTransition extends AbstractStateTransition {
```

Comment:
Why do we need the abstract class for Identity STs?  Other Identity STs will use Identity's public key as other STs.

Reply:
Oh.. My bad. TopUp will work in the same way.